entries/o/ox.ac.uk.json

```diff
@@ -0,0 +1,22 @@
+{
+  "University of Oxford": {
+    "domain": "ox.ac.uk",
+    "tfa": [
+      "totp",
+      "sms",
+      "custom-software",
+      "u2f",
```

Comment:
From what I read on https://help.it.ox.ac.uk/mfa-hardware-tokens#collapse2808966 it seems FIDO2 "passwordless" authentication is used instead of U2F/WebAuthn. Since only the hardware token is used we currently classify that as one factor of authentication.
```suggestion
```